cogs/giftdrop.py

```diff
@@ -350,7 +350,7 @@ async def join_command(self, ctx: commands.Context, *, nickname: str=''):
         results = test_username(nickname, ctx)
         if len(results) > 0:
             joined = ',\n'.join(results)
```

Comment:
Change `',\n'` to `'; '`

Reply:
I kept the \n because I felt like that it would be weird without it. However, I changed to the semicolon, it's better than the comma.